app/services/register_and_partner_api/sync_users.rb

```diff
@@ -38,9 +39,40 @@ def self.user_attributes_from(user_from_api)
         register_and_partner_id: user_from_api.attributes[:id],
         full_name: user_from_api.attributes[:attributes][:full_name],
         email: user_from_api.attributes[:attributes][:email],
-        cip: "UCL",
+        user_type: user_from_api.attributes[:attributes][:user_type],
+        cip: user_from_api.attributes[:attributes][:core_induction_programme],
       }
     end
-    private_class_method :sync_users, :user_attributes_from
+
+    def self.set_core_induction_programme(attributes, profile)
+      profile.core_induction_programme = CoreInductionProgramme.find_by(
+        name: determine_core_induction_programme(attributes[:cip]),
+      )
+    end
+
+    def self.determine_core_induction_programme(cip_name)
+      case cip_name
+      when "ambition"
+        "Ambition Institute"
+      when "edt"
+        "Education Development Trust"
+      when "teach_first"
+        "Teach First"
+      when "ucl"
+        "UCL"
+      end
+    end
+
+    def self.set_profile_type(user, user_type)
```

Comment:
`set_profile_type` is not a great name - the method actually returns us a profile based on user type.

`find_or_initialize_profile` would be a bit better :)